index.js

```diff
@@ -59,6 +59,7 @@ class ClinicDoctor extends events.EventEmitter {
 
     let NODE_PATH = path.join(__dirname, 'injects')
     // use NODE_PATH to work around issues with spaces in inject path
+    /* istanbul ignore if: spaces in path dependes on install location */
```

Comment:
https://github.com/nearform/node-clinic-doctor/pull/246 added a test for this `if`, so this can be un-ignored

Reply:
Thanks